orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/DisableServerGroupTaskSpec.groovy

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.tasks.servergroup
+
+import com.netflix.spinnaker.orca.clouddriver.FeaturesService
+import com.netflix.spinnaker.orca.clouddriver.utils.OortHelper
+import com.netflix.spinnaker.orca.pipeline.model.Pipeline
+import com.netflix.spinnaker.orca.pipeline.model.PipelineStage
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import spock.lang.Specification
+import spock.lang.Subject
+
+class DisableServerGroupTaskSpec extends Specification {
+
+  OortHelper oortHelper = Mock(OortHelper)
+  FeaturesService featuresService = Mock(FeaturesService)
+
+  Map otherServerGroup
+  Map targetServerGroup
+  Map context
+  Stage stage
+  List<Map> clusterTags
+
+  @Subject
+  DisableServerGroupTask task = new DisableServerGroupTask(oortHelper: oortHelper, featuresService: featuresService)
+
+  void setup() {
+    targetServerGroup = [
+      account: "test",
+      region : "us-east-1",
+      name   : "app-foo-v001"
+    ]
+    otherServerGroup = [
+      account   : "test",
+      region    : "us-east-1",
+      name      : "app-foo-v000",
+      isDisabled: true
+    ]
+    context = [
+      serverGroupName: "app-foo-v001",
+      account        : "test",
+      cloudProvider  : "aws",
+      region         : "us-east-1"
+    ]
+    stage = new PipelineStage(new Pipeline(), "disable server group", context)
+    clusterTags = [[ tags: [ [name: "spinnaker:no-disable-last-cluster", value: true]]]]
```

Comment:
what is the difference between `spinnaker:traffic-protection` in TrafficGuard and `spinnaker:no-disable-last-cluster` here ?